src/stac_api_benchmark/__main__.py

```diff
@@ -188,11 +194,13 @@ async def run(config: query.BenchmarkConfig) -> None:
 
 async def run_sort(config: query.BenchmarkConfig, field: str, direction: str) -> None:
     logger.info(f"Running sort {field} {direction}")
-    result = await query.sorting(
-        config,
-        sortby=[query.sortby(field, direction)],  # noqa
-    )
-    logger.info(f"Results: sort {field} {direction} : {result[1]:.2f}s")
+    for collection in config.collections:
+        result = await query.sorting(
+            config=config,
+            collection=collection,
+            sortby=[query.es_sortby(field, direction)],  # noqa
+        )
```

Comment:
Can (/do you want to) fire off all of these queries before `await`-ing them? Forgive me if this isn't how async in Python works, but I know in my experience if I want them all to go in parallel I need to spawn inside the loop then await on the handles.

Reply:
You're right about how that works. I wanted these to just be run one at a time, since they're more testing the individual sorting response times rather than concurrency.  These should probably be run like 10 times or something to get more than one data point for each one, but they all timeout right now anyway in PC.